tasklist/client/e2e/pages/FormJSDetailsPage.ts

```diff
@@ -50,13 +51,33 @@ class FormJSDetailsPage {
     this.selectDropdown = this.form.getByText('Select').last();
     this.checkbox = this.form.getByLabel('Checkbox');
     this.tagList = page.getByPlaceholder('Search');
+    this.dynamicListAddnewBtn = page.getByRole('button', {name: /add new/i});
   }
   async fillDate(date: string) {
     await this.dateInput.click();
     await this.dateInput.fill(date);
     await this.dateInput.press('Enter');
   }
 
+  async fillTextField(label: string, value: string) {
+    const locator = await this.getLocatorByLabel(label);
+    await locator.waitFor();
+    await locator.fill(value);
+  }
```

Comment:
We don't need this method, only create methods when more than one action is necessary.
This can simply become `page.getByLabel('Label value').fill('john')` in the test

It only makes sense to create methods for things like date pickers